non-wheel/httpcrtsrv/httpsrv.py

```diff
@@ -0,0 +1,444 @@
+    order allow,deny
+    allow from all
+    SetHandler cgi-script
+    Options +ExecCGI
+</Directory>
+#End XML Toolkit http settings
+
+# keep access logs 10 days, error logs 10 days, FastCGI logs 10 days
+LogMaint /www/%s/logs/access_log 10 0
+LogMaint /www/%s/logs/error_log 10 0
+LogMaint /www/%s/logs/error_zfcgi 10 0
+
+# Maintain Logs at 3 am (0 = midnight, 23 = 11 pm, etc)  
+# Set for an hour when the server is active (i.e. not during an IPL or backup)  
+LogMaintHour 3
+
+IncludeOptional /www/%s/conf/apache-sites/*.conf
+''' % (port, name, name, name, name, name, name)
+
+
+def get_example_vhost_conf(name, port):
+    return '''
+<VirtualHost *:%s>
+    ServerName %s.domain.com
+    DocumentRoot /www/%s/example-project/htdocs
+
+    <Directory "/www/%s/example-project/htdocs">
+        Options Indexes FollowSymLinks
+        AllowOverride All
+        Require all granted
+        DirectoryIndex index.php index.html
+    </Directory>
+
+
+    ErrorLog "/www/%s/example-project/logs/error_log"
+    CustomLog "/www/%s/example-project/logs/access_log" common
+</VirtualHost>
+''' % (port, name, name, name, name, name)
+
+
+def create(name, conf, port):
+    path = '/www/%s' % name
+    verification_text = """
+    /www/
+        %s/
+            conf/
+                apache-sites/
+                    example-vhost.conf
+                httpd.conf
+            example-project/
+                conf/
+                    %s.conf
+                htdocs/
+                    index.html
+                logs/
+            htdocs/
+                index.html
+            logs/
+""" % (name, name)
+
+    verification_text += "Does the above folder structure look correct? (Y/n) "
+
+    verified = input(verification_text) or 'Y'
+
+    if verified in ['y', 'Y', 'yes', 'Yes', 'YES']:
+        print('conf: %s' % conf)
+        print('name: %s' % name)
+        print('port: %s' % port)
+        os.mkdir('%s' % path)
+        os.mkdir('%s/conf' % path)
+        os.mkdir('%s/logs' % path)
+        os.mkdir('%s/htdocs' % path)
+        index_file = open("%s/htdocs/index.html" % path, "w+")
+        index_file.write(get_index_html(name))
+
+        conf_file = open("%s/conf/httpd.conf" % path, "w+")
+        fastcgi_conf_file = open("%s/conf/fastcgi.conf" % path, "w+")
+        fastcgi_conf_twn_file = open("%s/conf/fastcgi.conf.twn" % path, "w+")
+        fastcgi_dynamic_conf_file = open("%s/conf/fastcgi_dynamic.conf" % path, "w+")
+        fastcgi_http_add_conf_file = open("%s/conf/fastcgi_http_add.conf" % path, "w+")
+        fastcgi_conf_file.write(get_fastcgi_conf(name))
+        fastcgi_conf_twn_file.write(get_fastcgi_conf_twn(name))
+        fastcgi_dynamic_conf_file.write(get_fastcgi_dynamic_conf(name))
+        fastcgi_http_add_conf_file.write(get_fastcgi_http_add_conf(name))
+        fastcgi_conf_file.close()
+        fastcgi_conf_twn_file.close()
+        fastcgi_dynamic_conf_file.close()
+        fastcgi_http_add_conf_file.close()

```

Comment:
I'd recommend for the above to open the files using the `with` statement (as a context manager) so you don't have to manually close them:

```python
with open("%s/conf/fastcgi.conf" % path, "w+") as fastcgi_conf_file:
    fastcgi_conf_file.write(get_fastcgi_conf(name))
```